tests/common/plugins/conditional_mark/issue.py

```diff
@@ -56,6 +56,9 @@ def get_cred(self):
                     github_creds = creds.get(self.NAME, {})
                     self.user = github_creds.get('user', '')
                     self.api_token = github_creds.get('api_token', '')
+                else:
+                    self.user = os.environ["USER_NAME"]
+                    self.api_token = os.environ["API_TOKEN"]
```

Comment:
Improve the variable names to `GITHUB_USERNAME` and `GITHUB_API_TOKEN` to make them more explicit?